packages/@okta/vuepress-site/docs/guides/mobile-idx-sdk-overview/main/index.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Overview of the mobile Identity Engine SDK
+---
+
+<div class="oie-embedded-sdk">
+
+Design the implementation of the sign-in flow for your mobile app by understanding the
+representation of the flow, and with examples of the common parts.
+
+<ApiLifecycle access="ie" /><br>
+
+## Introduction
+
+Integrate the Okta sign-in flow into your app in one of two ways: redirect the request to
+Okta which handles the flow in its own window, or handle each step in the flow by
+building the appropriate user interface elements, capturing the information, and updating
+Okta.
+
+This guide is an overview of building your own user interface. For information on the redirect model, see <StackSnippet snippet="redirectquickstart" inline />.
+
+## Sign-in flow
+
+Okta supports many ways of authenticating the identity of a user during the sign-in flow.
+An Okta org administrator creates _policies_, different mixes of authentication methods
```

Comment:
Re the emphasis, this is Susan approved (see the replies to your comments above) so it's fine.

As to the wording, I used the structure above because I'm introducing a potentially new term. I use emphasis and the comma to indicate "this is a new term that you may not know", "here's a simple definition."